examples/bert_ptq_cpu/bert_config.json

```diff
@@ -3,15 +3,28 @@
     "input_model":{
         "type": "PyTorchModel",
         "config": {
-            "model_path": null,
-            "model_storage_kind": "folder",
-            "model_loader": "load_pytorch_origin_model",
-            "model_script": "user_script.py",
+            "hf_config": {
+                "model_name": "Intel/bert-base-uncased-mrpc",
+                "task": "text-classification",
+                "dataset": {
```

Comment:
does HFConfig need to be updated for the dataset?

Reply:
`HFConfig` is a `dict` which can accept related config where we did not restrict the key. Also I updated it in Olive docs.
![Uploading image.png…]()